fjord/feedback/views.py

```diff
@@ -72,11 +72,18 @@ def thanks(request):
             pass
 
     if feedback:
-        product = feedback.product
+        product_db_name = feedback.product
         suggestions = get_suggestions(feedback, request)
 
-    return render(request, 'feedback/thanks.html', {
-        'product': product,
+    product = models.Product.objects.get(db_name=product_db_name)
+
+    template = 'feedback/thanks.html'
+
+    if product.slug in PRODUCT_OVERRIDE:
+      template = PRODUCT_OVERRIDE[product.slug]['thanks_template']
```

Comment:
Nit: We use four spaces--not two.

Also, instead of doing this here, you should pass the thank you page in via the args.